eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -48,24 +77,35 @@ jobs:
         parameters:
           DependenciesVariableName: vcpkg.deps
 
-      - template: /eng/pipelines/templates/steps/cmake-build.yml
-        parameters:
-          BuildArgs: $(build.args)
+      - pwsh: |
+           Write-Host ("Powershell Module Path:" + $Env:PSModulePath)
+           Get-Module -ListAvailable
+           Write-Host "##vso[task.setvariable variable=VCPKG_ROOT]$(Agent.TempDirectory)/vcpkg"
+        displayName: "List available Powershell Modules"
 
       - template: /eng/common/TestResources/deploy-test-resources.yml
         parameters:
           ServiceDirectory: '.'
 
+      - pwsh: |
+           Write-Host "##vso[task.setvariable variable=VCPKG_ROOT]$(Agent.TempDirectory)/vcpkg"
+        displayName: "List VCPKG env vars"
```

Comment:
The `displayName` is not accurate here. We already set `VCPKG_INSTALLATION_ROOT` in vcpkg.yml. If `VCPKG_INSTALLATION_ROOT` can be used instead I recommend we do that instead of duplicating the variables.

Reply:
lets see what happens when we delete